tests/funcs.cc

```diff
@@ -1,24 +1,24 @@
 #include <glog/logging.h>
 
-#include "gtest/gtest.h"
 #include "kvstore/db.h"
 #include "kvstore/store.h"
 #include "warble/functions.h"
+#include "gtest/gtest.h"
 
 class FakeDB : public Database {
- public:
+public:
```

Comment:
from https://google.github.io/styleguide/cppguide.html#Class_Format
> The public:, protected:, and private: keywords should be indented one space.

Reply:
I think clang-format made this change. I will fix this.